@yusuke-arai さん
値が変わった箇所について、変わっていることが妥当であれば、exp 値の定数を更新していただいて大丈夫です。
既存挙動の破壊検知ができた、ということだけでも有益なテストでした。
２つの居室に変更があることが妥当かどうか、になります。

E           AssertionError: 
E           Arrays are not almost equal to 2 decimals
E           
E           Mismatched elements: 2 / 5 (40%)
E           Max absolute difference among violations: 0.46702219
E           Max relative difference among violations: 0.02173207
E            ACTUAL: array([[19.63],
E                  [20.87],
E                  [21.02],...
E            DESIRED: array([[19.63],
E                  [21.29],
E                  [21.49],...

--- a/src/tests/carryover_heat/test_4_2_formula_46_48.py
+++ b/src/tests/carryover_heat/test_4_2_formula_46_48.py
@@ -47,7 +47,7 @@ def test_過剰熱量繰越を考慮した室温_居室_式46():
     assert Theta_HBR_i.shape == (5, 1), "Theta_HBR_iの次数が想定外"
 
     # NOTE: 資料ではキャップされていないが、実装ではキャップされている
-    exp_Theta_HBR_i = np.array([19.63, 21.29, 21.49, 19.97, 19.64]).reshape(-1,1)
+    exp_Theta_HBR_i = np.array([19.63, 20.87, 21.02, 19.97, 19.64]).reshape(-1,1)
 
     np.testing.assert_almost_equal(Theta_HBR_i, exp_Theta_HBR_i, decimal=2), "Theta_HBR_iの計算がおかしい"
     # assert つかないの間違えやすいので注意

